developer/frameworks/articles/05-developing-fragments/00-developing-fragments-intro.markdown

```diff
@@ -0,0 +1,72 @@
+# Page Fragments [](id=developing-fragments)
+
+Page Fragments take your design vision and accurately realize it on a web page. 
+To do this, developers are given a space where they have a "blank slate."  You 
+have three tools at your disposal to accomplish this:
+
+**HTML**: The markup of the fragment. Fragments use standard HTML with 
+   special tags to add dynamic behavior.
+
+**CSS**: CSS styles and positions the fragment's markup. 
+
+**JavaScript**: JavaScript provides dynamic behavior to the fragment.
+
+The HTML, CSS, and JavaScript are all completely standard, just like anywhere
+else on the web, but are also enhanced with Liferay-specific features. You can
+specify text and images as editable, as well as providing for "rich text": that
```

Comment:
we also provide now editable elements of type link, I'll add a comment in the fragment specific tags section, but it would be good to mention it here too